[2023-03-15T02:13:04Z ERROR wgpu::backend::direct] Handling wgpu errors as fatal by default
thread 'main' panicked at 'wgpu error: Validation Error

Caused by:
    In Device::create_sampler
    invalid lod clamp lod_min_clamp:-100 lod_max_clamp:100, must satisfy lod_min_clamp >= 0 and lod_max_clamp >= lod_min_clamp

', C:\Users\thisi\.cargo\registry\src\github.com-1ecc6299db9ec823\wgpu-0.15.1\src\backend\direct.rs:3024:5
stack backtrace:
   0: std::panicking::begin_panic_handler
             at /rustc/69f9c33d71c871fc16ac445211281c6e7a340943/library\std\src\panicking.rs:575
   1: core::panicking::panic_fmt
             at /rustc/69f9c33d71c871fc16ac445211281c6e7a340943/library\core\src\panicking.rs:65
   2: wgpu::backend::direct::default_error_handler
             at C:\Users\thisi\.cargo\registry\src\github.com-1ecc6299db9ec823\wgpu-0.15.1\src\backend\direct.rs:3024
   3: core::ops::function::Fn::call<void (*)(enum2$<wgpu::Error>),tuple$<enum2$<wgpu::Error> > >
             at /rustc/69f9c33d71c871fc16ac445211281c6e7a340943\library\core\src\ops\function.rs:78
   4: alloc::boxed::impl$47::call<tuple$<enum2$<wgpu::Error> >,dyn$<wgpu::UncapturedErrorHandler<assoc$<Output,tuple$<> > > >,alloc::alloc::Global>
             at /rustc/69f9c33d71c871fc16ac445211281c6e7a340943\library\alloc\src\boxed.rs:2001
   5: wgpu::backend::direct::ErrorSinkRaw::handle_error
             at C:\Users\thisi\.cargo\registry\src\github.com-1ecc6299db9ec823\wgpu-0.15.1\src\backend\direct.rs:3010
   6: wgpu::backend::direct::Context::handle_error<enum2$<wgpu_core::resource::CreateSamplerError> >
             at C:\Users\thisi\.cargo\registry\src\github.com-1ecc6299db9ec823\wgpu-0.15.1\src\backend\direct.rs:296
   7: wgpu::backend::direct::impl$7::device_create_sampler
             at C:\Users\thisi\.cargo\registry\src\github.com-1ecc6299db9ec823\wgpu-0.15.1\src\backend\direct.rs:1333
   8: wgpu::context::impl$5::device_create_sampler<wgpu::backend::direct::Context>
             at C:\Users\thisi\.cargo\registry\src\github.com-1ecc6299db9ec823\wgpu-0.15.1\src\context.rs:2240
   9: wgpu::Device::create_sampler
             at C:\Users\thisi\.cargo\registry\src\github.com-1ecc6299db9ec823\wgpu-0.15.1\src\lib.rs:2164
  10: voxl::gfx::render::texture::Texture::create_depth_texture
             at .\src\gfx\render\texture.rs:114
  11: voxl::gfx::render::renderer::impl$2::new::async_fn$0
             at .\src\gfx\render\renderer.rs:344
  12: core::future::from_generator::impl$1::poll<enum2$<voxl::gfx::render::renderer::impl$2::new::async_fn_env$0> >
             at /rustc/69f9c33d71c871fc16ac445211281c6e7a340943\library\core\src\future\mod.rs:91
  13: voxl::run::async_fn$0
             at .\src\main.rs:35
  14: core::future::from_generator::impl$1::poll<enum2$<voxl::run::async_fn_env$0> >
             at /rustc/69f9c33d71c871fc16ac445211281c6e7a340943\library\core\src\future\mod.rs:91
  15: pollster::block_on<core::future::from_generator::GenFuture<enum2$<voxl::run::async_fn_env$0> > >
             at C:\Users\thisi\.cargo\registry\src\github.com-1ecc6299db9ec823\pollster-0.3.0\src\lib.rs:128
  16: voxl::main
             at .\src\main.rs:95
  17: core::ops::function::FnOnce::call_once<void (*)(),tuple$<> >
             at /rustc/69f9c33d71c871fc16ac445211281c6e7a340943\library\core\src\ops\function.rs:251
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
error: process didn't exit successfully: `target\debug\voxl.exe` (exit code: 101)